### PR TITLE
avatar added

### PR DESCRIPTION
**Pull request by dipen2000**
Just created ann avatar component review it and let me know.

### CODE REVIEW

**Comment by dipu6812** [comment]
I am just testing

File: index.html [hunk 1]
```diff
@@ -9,6 +9,11 @@
 </head>
 <body>
     <h1>Hello</h1>
-    <p>I am Dipen</p>
+    <div>1</div>
+    <div>2</div>
+    <div>3</div>
+    <div>4</div>
+    <div>5</div>
+    <div>6</div>
```
Comment (dipu6812) [inline index.html:17]: good

Comment (dipu6812) [inline index.html:17]: only this line

Comment (dipen2000) [reply index.html:17]: ok

File: index.html [hunk 2]
```diff
@@ -9,6 +9,11 @@
 </head>
 <body>
     <h1>Hello</h1>
-    <p>I am Dipen</p>
```
Comment (dipu6812) [inline index.html:12]: You should not have removed this.

File: index.html [hunk 3]
```diff
@@ -9,6 +9,11 @@
...
+    <div>1</div>
+    <div>2</div>
+    <div>3</div>
```
Comment (dipu6812) [inline index.html:14]: remove these two line.

File: index.html [hunk 4]
```diff
@@ -9,6 +9,10 @@
...
+    <div>4</div>
+    <div>changes done</div>
```
Comment (dipu6812) [inline index.html:16]: changes done this is awesome.

File: index.html [hunk 5]
```diff
@@ -9,6 +9,10 @@
 </head>
 <body>
     <h1>Hello</h1>
-    <p>I am Dipen</p>
+    <div>1</div>
+    <div>2</div>
+    <div>3</div>
```
Comment (dipu6812) [inline index.html:14]: Remove these two as well.

**Review by dipu6812 — CHANGES_REQUESTED** [review]
Do the things according to the comments then you are done.

**Review by dipu6812 — APPROVED** [review]
OK this looks good , go ahead and merge it.

**Pull request merged**